File: C++/hello.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    for (int i = 0; i < 100; i++)
    {
        string hello = "C++: Hello World!";
        cout << hello << " No." << i << endl;
    }
    return 0;
}

//  g++ hello.cpp && ./a.out

File: C#/Hello.cs
using System;

namespace HelloWorld
{
    class Program
    {
        static void Main(string[] args)
        {
            for(int i=0;i<100;i++)
            {
                string text = String.Format("C#: Hello, World! No.{0}",i);
                System.Console.WriteLine(text);
            }
        }
    }
} 

// dotnet new console
// dotnet build && dotnet bin/Debug/net5.0/hello.dll
// 